packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1049,6 +1081,62 @@ export class AttributesController extends ConsumptionBaseController {
         }
     }
 
+    private async transmitDefault(attribute: LocalAttribute): Promise<void> {
```

Comment:
1. If find the name confusing as 'transmit' means to send something over the wire. Did you meant to call it 'transfer'? How about something simple as `setOtherAttributeAsDefault`?
2. Your other `...default` methods return the new default attribute, while this one returns void. I understand that you don't need the result anywhere but I find it easier to understand when functions which do similar things also have similar signatures.

Reply:
1. Yes, 'transfer' seems to describe it more appropriately. Regarding `setOtherAttributeAsDefault` I have still the same concerns as already discussed above. If the Attribute I delete wasn't the former default Attribute, we won't set another Attribute as default. This is a point I want to express with `transfer`.
2. I see where you are coming from, but I'm having trouble applying this here. In the other `...default` methods you have the same (adjusted) Attribute as input and output. However, here we an Attribute that could be of any kind. If it wasn't the former Attribute, we would simply return it unchanged. If it was and we transfer the default, would we then return the former default Attribute, which is not a default Attribute anymore? Or would we return the new default Attribute, which is contrary to the case, that the input Attribute was no a default Attribute. Thus, I think the solution with least complexity is to not return anything at all.